app/views/layouts/_footer.html.haml

```diff
@@ -13,9 +13,9 @@
               %i.fas.fa-info
               About
           %li
-            = link_to(gallery_path) do
+            = link_to('https://www.flickr.com/groups/366038@N24/pool/', target: :_blank) do
```

Comment:
```suggestion
            = link_to('https://www.flickr.com/groups/hackweek/') do
```

Reply:
Fiddling with peoples tabs is not really a thing in 2022 anymore ;-)

Reply:
removed